src/servers/trigger.ts

```diff
@@ -255,7 +261,13 @@ const main = () => {
 const healthCheck = () => {
   const app = express()
   const port = 8888
-  app.get("/healthz", (req, res) => res.sendStatus(200))
+  app.get("/healthz", async (_req, res) => {
+    const isMongoAlive = mongoose.connection.readyState === 1
+    const isRedisAlive = (await redis.ping()) === "PONG"
+    const statuses = Object.values(lndStatus)
```

Comment:
maybe this should be refactored so that the other servers has the code/use of lndStatus.

Reply:
but should we check only main lnd? or both should be active?

Reply:
at least 1

Reply:
one thing I'm not sure how this would work: if one lnd were to restart, but not trigger, would trigger be able to reconnect to it?

Reply:
yep, it works, you can test locally with:

- Run docker stop <galoy-backend-lnd2-1 container id>
- GET http://localhost:8888/healthz (should receive 503)
- Run docker start <galoy-backend-lnd2-1 container id>
- GET http://localhost:8888/healthz (should receive 200)

Reply:
what works exactly? trigger would restart?

Reply:
we don't want trigger to continuously restart if one lnd is continuously failing

Reply:
I changed the validation to some, so trigger will not restart until both are offline (or if is not able to connect)

Reply:
Refactor done for exporter, graphql and trigger but I couldn't add `is_ready` (Wallet Is Ready For RPC Calls) because is not supported on LND 0.13.4 and below